### DISCUSSION TITLE
[Player Support] foobar2000

### DISCUSSION THREAD

**Discussion by Bestfast** [Syncplay/syncplay#660] [Ideas]
As title suggests, I think there are no current good ways to enjoy music together with friends. The closest we can get is probably Jellyfin's group with a music library, or streaming a media player on VoIP programs, but that would be awful quality wise. Instead of going by filename, you could simply behave with a track artist and title.
https://wiki.hydrogenaud.io/index.php?title=Foobar2000:Commandline_Guide this could be of help.

**Comment by Et0h** [comment]
If you want to enjoy music via Sycplay you can use Syncplay with the mpv or mpv.net media players, either listening to music on YouTube via youtube-dl/yt-dlp or locally stored music files.

I've had a quick look, and it doesn't look like foobar2000 provides an API that would allow Syncplay to control and track it.

If someone wanted to write a foobar2000 plugin then it might be possible for someone to code it to use the API Syncplay uses for VLC, but implementing such a thing would be a big commitment for someone to take on.
